src/precompile.js

```diff
@@ -56,25 +56,7 @@ function precompile(input, opts) {
     else if(pathStats.isDirectory()) {
         var templates = [];
 
-        function addTemplates(dir) {
```

Comment:
Why? It's a more functional style and I think it's perfectly fine if abstracting out that logic makes sense. I wouldn't worry about refactoring this file because I'm already refactoring it with a new API (there's an issue but I forget the number)